cluster/addons/addon-manager/Makefile

```diff
@@ -15,10 +15,10 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v8.8
-KUBECTL_VERSION?=v1.10.7
+VERSION=v8.9
+KUBECTL_VERSION?=v1.10.9
```

Comment:
@MrHohn does this version of kubectl still make sense? or should we bump to 1.11.x?

Reply:
Thanks for adding me, bumping to 1.11.x sounds good to me.

Reply:
updated to kubectl v1.11.3.